### PR TITLE
PRM-990 Headers update

### PR DESCRIPTION
**Pull request by AndrewRButcher**

### CODE REVIEW

**Review by LewisMitchell — CHANGES_REQUESTED** [review]

File: it/stubs/PenaltiesStub.scala [hunk 1]
```diff
@@ -18,23 +18,37 @@ package stubs
...
 
 object PenaltiesStub {
   private val appealUriForLSP = "/penalties/appeals-data/late-submissions"
   private val appealUriForLPP = "/penalties/appeals-data/late-payments"
   private val fetchReasonableExcuseUri =
     "/penalties/appeals-data/reasonable-excuses"
-  private val submitAppealUri = (isLPP: Boolean, penaltyId: String) =>
-    s"/penalties/appeals/submit-appeal?enrolmentKey=HMRC-MTD-VAT~VRN~123456789&isLPP=$isLPP&penaltyId=$penaltyId"
+//  private val submitAppealUri = (isLPP: Boolean, penaltyId: String, correlationId: String) =>
+//    s"/penalties/appeals/submit-appeal?enrolmentKey=HMRC-MTD-VAT~VRN~123456789&isLPP=$isLPP&penaltyId=$penaltyId&correlationId=$correlationId"
```
[the remark this answers is not in the document]
Comment (AndrewRButcher) [reply it/stubs/PenaltiesStub.scala]: Done ✅

File: it/stubs/PenaltiesStub.scala [hunk 2]
```diff
@@ -18,23 +18,37 @@ package stubs
...
+  private val submitAppealQueryParams = (isLPP: Boolean, penaltyId: String) => Map[String, StringValuePattern](
+  ("enrolmentKey" -> containing("HMRC-MTD-VAT~VRN~123456789")),
+  ("isLPP" -> containing(isLPP.toString)),
+  ("penaltyId" -> containing(penaltyId)),
+  ("correlationId" -> matching(".*"))
+)
+
+  val jmap = new java.util.HashMap[String, String]()
+  jmap.put("fname", "Alvin")
+  jmap.put("lname", "Alexander")
```
[the remark this answers is not in the document]
Comment (AndrewRButcher) [reply it/stubs/PenaltiesStub.scala]: Done ✅

File: it/stubs/PenaltiesStub.scala [hunk 3]
```diff
@@ -18,23 +18,37 @@ package stubs
...
+//  private val submitAppealUri = (isLPP: Boolean, penaltyId: String, correlationId: String) =>
+//    s"/penalties/appeals/submit-appeal?enrolmentKey=HMRC-MTD-VAT~VRN~123456789&isLPP=$isLPP&penaltyId=$penaltyId&correlationId=$correlationId"
   private val fetchOtherPenalties = (penaltyID: String) =>
     s"/penalties/appeals/multiple-penalties-in-same-period?enrolmentKey=HMRC-MTD-VAT~VRN~123456789&penaltyId=$penaltyID&isLPP=false"
 
+  private val submitAppealUri = "/penalties/appeals/submit-appeal"
+  private val submitAppealQueryParams = (isLPP: Boolean, penaltyId: String) => Map[String, StringValuePattern](
+  ("enrolmentKey" -> containing("HMRC-MTD-VAT~VRN~123456789")),
+  ("isLPP" -> containing(isLPP.toString)),
+  ("penaltyId" -> containing(penaltyId)),
```
[the remark this answers is not in the document]
Comment (AndrewRButcher) [reply it/stubs/PenaltiesStub.scala]: changed to use equalTo as matching is for regexes

File: conf/application.conf [hunk 4]
```diff
@@ -184,11 +184,6 @@ mongodb {
   ttl = 1 hour
 }
 
-pega {
-    bearerToken = "placeholder"
-    environment = "environmentValue"  # environmentValue for QA,staging and production: QA = ist0, staging = clone, production = live
```
[the remark this answers is not in the document]
Comment (AndrewRButcher) [reply conf/application.conf:189]: PR raised https://github.com/hmrc/app-config-production/pull/13677

**Review by LewisMitchell — APPROVED** [review]

**Comment by LewisMitchell** [comment]
Waiting on https://github.com/hmrc/penalties/pull/93

**Pull request merged**